app/setting/controllers/setting-users-controller.js

```diff
@@ -64,6 +66,10 @@ function (
         return false;
     };
 
+    $scope.addUser = function () {
+        $location.path('/settings/users/create');
```

Comment:
We should really never be doing this. This is clearly a link, so use a link not a button and let browser navigation take car of it. If the CSS doesn't work as a button, thats a bug in the PL.